test/fuzz/common.go

```diff
@@ -79,10 +85,10 @@ func GenerateNodeSelectorTerm(cf *fuzz.ConsumeFuzzer, term *corev1.NodeSelectorT
 	}
 
 	if len(term.MatchExpressions) == 0 {
-		term.MatchExpressions = make([]corev1.NodeSelectorRequirement, rand.Intn(2)+1)
+		term.MatchExpressions = make([]corev1.NodeSelectorRequirement, r.Intn(2)+1)
```

Comment:
plz replace 2 with a constant and comment the meaning of the constant

Reply:
> plz replace 2 with a constant and comment the meaning of the constant

done